lib/install/config/webpack/shared.js

```diff
@@ -4,16 +4,18 @@ const webpack = require('webpack')
 const path = require('path')
 const process = require('process')
 const glob = require('glob')
+const ManifestPlugin = require('webpack-manifest-plugin')
 const extname = require('path-complete-extname')
+const { webpacker } = require('../../package.json')
```

Comment:
I'd do `{ config }` instead and then just reference things directly off that object rather than assigning a bunch of consts.